src/inmanta/agent/config.py

```diff
@@ -132,6 +132,17 @@
     is_time,
 )
 
+agent_get_resource_backoff = Option(
+    "config",
+    "agent-get-resource-backoff",
+    5,
```

Comment:
are we going to change the default on master?

Reply:
Yes. I will create a separate PR for that as we don't want to change the default on the iso6 branch. This one goes into `master` and `iso6`.